mate_bot/commands/balance.py

```diff
@@ -16,7 +16,9 @@ class BalanceCommand(BaseCommand):
     """
 
     def __init__(self):
-        super().__init__("balance", "Shows a user's balance.")
+        super().__init__("balance", "The `/balance` command prints the current account balance of an "
+                                    "account. If no account name is provided the balance of the "
+                                    "sender is given.")
```

Comment:
This is useful and looks nice. Add a comma between `is provided` and `the balance`, as it improves readability.